libs/src/services/discoveryProvider/constants.js

```diff
@@ -1,6 +1,7 @@
 module.exports.DISCOVERY_PROVIDER_TIMESTAMP = '@audius/libs:discovery-provider-timestamp'
 module.exports.DISCOVERY_SERVICE_NAME = 'discovery-provider'
 module.exports.UNHEALTHY_BLOCK_DIFF = 15
+module.exports.DEBUG_UNHEALTHY_BLOCK_DIFF = 100
```

Comment:
why do we need this?

Reply:
I was seeing issues with DP falling behind and failing operations between adding users and upgrading them. I could add wait time to the test for DP to catch up, but then it's a wait between every user upgrade operation, significantly slowing down the test setup at the beginning. But maybe that's the best approach to minimize protocol changes

Reply:
my take would be to make the test and tooling fit the system, not vice versa. that's also why i tend to be a bit opinionated about the health check as well. i'm not concerned about protocol changes themselves. What concerns me is having levers we can pull in either direction. that seems dangerous given that the final outcome we're concerned with is the behavior of the protocol

Reply:
I buy that argument, in particular wrt to the block diff check. I'm a bit more iffy about trying to put the health check back though, if that would necessitate retries and make the test flakier. It seems like the correct fix would be to figure out *why* the health check is flaky to begin with, but I don't think that's in scope for this task. I'll remove the block diff and try to make the test accommodate it, but I'd rather not bring in the health check until we can set aside some time to figure out why it doesn't come up consistently.